README.md

```diff
@@ -77,6 +78,7 @@ _Note_: The descriptions use the npm package parameters, but they also apply to
 | `apiKey` | `api_key` | Datadog API Key, only needed when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set. For more information about getting a Datadog API key, see the [API key documentation][8]. |
 | `apiKmsKey` | `api_kms_key` | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set, and you are using KMS encryption. |
 | `enableDatadogTracing` | `enable_datadog_tracing` | Enable Datadog tracing on your Lambda functions. Defaults to `true`. |
+| `enableDatadogLogs` | `enable_datadog_logs` | Enable Datadog logs on your Lambda functions. Defaults to `true`. |
```

Comment:
Per my comment on your [other PR](https://github.com/DataDog/serverless-plugin-datadog/pull/146), I don't think this will apply when using the Datadog Forwarder instead of the Datadog Lambda Extension. Let me know if you want more info about this! 🙂

Reply:
Thanks! I don't think I'm getting github notifications via email yet, so I didn't see that comment yesterday